custom-recipes/neo4j-export-nodes/recipe.json

```diff
@@ -110,6 +110,28 @@
             "description": "",
             "defaultValue": false
         },
+        {
+            "label": "Handling Null Values",
```

Comment:
This new section should go at the bottom (below the `Export parameters function`)